tap_typeform/streams.py

```diff
@@ -175,9 +175,9 @@ def sync_form(atx, form_id, start_date, end_date):
 
                 answers_data_rows.append({
                     "landing_id": row['landing_id'],
```

Comment:
This is what it should look like: `"landing_id": row.get('landing_id'),`

Reply:
Perfect--made that change too!